File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Application/Common/Identity/Models/SignInDetails.cs
namespace Feedback.Analyzer.Application.Common.Identity.Models;

/// <summary>
/// Represents sign in details by email
/// </summary>
public class SignInDetails
{
    /// <summary>
    /// Gets sign in email address
    /// </summary>
    public string EmailAddress { get; set; } = default!;
    
    /// <summary>
    /// Gets sign in password
    /// </summary>
    public string Password { get; set; } = default!;

    /// <summary>
    /// Gets sign extended active time
    /// </summary>
    public bool RememberMe { get; set; }
}

File: src/Insight.Box.FrontEnd/src/modules/accounts/models/SignInDetails.ts


/**
 * Represents the details required for user sign-in.
 */
export class SignInDetails {
    /**
     * The email address of the user.
     */
    public emailAddress: string = '';

    /**
     * The password associated with the user's account.
     */
    public password: string = '';

    /**
     * Constructs a new instance of SignInDetails class.
     * Initializes email address and password to empty strings by default.
     */
    constructor() {
        this.emailAddress = "";
        this.password = "";
    }
}
